website/content/en/preview/concepts/nodeclaims.md

```diff
@@ -0,0 +1,276 @@
+       "controllerGroup":"karpenter.sh","controllerKind":"NodeClaim","NodeClaim":
+       {"name":"default-sfpsl"},"namespace":"","name":"default-sfpsl","reconcileID":
+       "9c9dbc80-3f0f-43ab-b01d-faac6c29e979","provider-id":
+       "aws:///us-west-2b/i-08a3bf1cadb205c7e","instance-type":"c3.2xlarge","zone":
+       "us-west-2b","capacity-type":"spot","allocatable":{"cpu":"7910m",
+       "ephemeral-storage":"17Gi","memory":"13215Mi","pods":"58"}}
+    ```
+ 
+5. Gathers necessary metadata from the NodeClaim to check and setup the node.
+6. Checks that the Node exists, and prepares it for use.  This includes:
+    * Waiting for the instance to return a provider ID, instance type, zone, capacity type,
+      and an allocatable status, indicating that the instance is ready.
+    * Checking to see if the node has been synced, adding a finalizer to the node (this provides the same
+      termination guarantees that all Karpenter nodes have), passing in labels, and updating the node owner references.
+      Example of log messages at this stage:
+      ```
+      {"level":"INFO","time":"2024-06-22T02:24:52.642Z","logger":"controller","message":
+         "initialized nodeclaim","commit":"490ef94","controller":
+         "nodeclaim.lifecycle","controllerGroup":"karpenter.sh","controllerKind":
+         "NodeClaim","NodeClaim":{"name":"default-sfpsl"},"namespace":
+         "","name":"default-sfpsl","reconcileID":
+         "7e7a671d-887f-428d-bd79-ddf603290f0a",
+         "provider-id":"aws:///us-west-2b/i-08a3bf1cadb205c7e",
+         "Node":{"name":"ip-192-168-170-220.us-west-2.compute.internal"},
+         "allocatable":{"cpu":"7910m","ephemeral-storage":"18242267924",
+         "hugepages-2Mi":"0","memory":"14320468Ki","pods":"58"}}
+      ```
+7. Making sure that the Node is ready to use. This includes such things as seeing if resources are registered and start-up taints are removed.
+8. Checking the nodes for liveliness.
+
+At this point, the node is considered ready to go.
+
+Karpenter assumes that a NodeClaim will register within 15 minutes of being requested.
+Karpenter will not launch additional capacity while waiting for this.
+As they are booting up, NodeClaims are included in the scheduling simulation for both launch and termination flows.
+
+If a node doesn’t appear as registered after 15 minutes, the NodeClaim is deleted.
+Karpenter assumes there is a problem that isn’t going to be fixed.
+No pods will be deployed there. After the node is deleted, Karpenter will try to create a new NodeClaim.
+
+## NodeClaim drift and disruption
+
+Although NodeClaims play a role in replacing nodes that drift or have been disrupted,
+to a Karpenter user, NodeClaims are immutable and cannot be modified.
+NodeClaims should only be a tool for monitoring.
+
+For details on Karpenter disruption, see [Disruption]({{< ref "./disruption" >}}).
```

Comment:
What is the purpose of this section at the moment? It doesn't seem like it includes any real details about drift and disruption so I question if we should include it at all. If we wanted to include details on drift we could talk about how the NodeClaim is immutable, and when the state of the NodeClaim is no longer compatible with the desired state from the NodePool / NodeClass we mark the NodeClaim as drifted. I'm not sure we need to go into that detail here though, and would probably be better suited to the disruption docs (if we document it at all and don't just treat it as an implementation detail).